javaparser-core/src/main/java/com/github/javaparser/javadoc/Javadoc.java

```diff
@@ -106,8 +106,25 @@ public JavadocDescription getDescription() {
         return description;
     }
 
+    public JavadocBlockTag getBlockTag(int i) {
```

Comment:
Why we do not want to allow the direct modification of the list? Do we want to prevent certain changes?

Reply:
In my opinion direct modification wouldn't be bad, by introducing these additional methods (`getBlockTag(int)`, etc.) we are only cluttering the code without any added value.

We should probably then add a `getElements()` method in `JavadocDescription` for consistency as well.

Regarding modification:
Other classes, such as [SwitchStmt](https://github.com/matozoid/javaparser/blob/268ded9301f347f74267f57927a34e29a6bee7a1/javaparser-core/src/main/java/com/github/javaparser/ast/stmt/SwitchStmt.java#L89) follow the same approach with a visible getter to retrieve the underlying List...